docs/dasharo-tools-suite/releases.md

```diff
@@ -13,64 +13,95 @@ For details about our release process please read
 
 [newsletter]: https://newsletter.3mdeb.com/subscription/ttzqCq9fy
 
-## v1.1.0
+## v1.1.1 - 2023-02-20
+
+### Images
+
+* [USB bootable DTS v1.1.1 image](https://3mdeb.com/open-source-firmware/DTS/v1.1.1/dts-base-image-v1.1.1.wic.gz)
+* [sha256](https://3mdeb.com/open-source-firmware/DTS/v1.1.1/dts-base-image-v1.1.1.wic.gz.sha256)
+* [sha256.sig](https://3mdeb.com/open-source-firmware/DTS/v1.1.1/dts-base-image-v1.1.1.wic.gz.sha256.sig)
+* [DTS v1.1.1 ISO image](https://3mdeb.com/open-source-firmware/DTS/v1.1.1/dts-base-image-v1.1.1.iso)
+* [sha256 ISO](https://3mdeb.com/open-source-firmware/DTS/v1.1.1/dts-base-image-v1.1.1.iso.sha256)
+* [sha256.sig ISO](https://3mdeb.com/open-source-firmware/DTS/v1.1.1/dts-base-image-v1.1.1.iso.sha256.sig)
+
+  See how to verify hash and signature on [this
+  video](https://youtu.be/RF-NYcZM9JI). It works the same way with ISO image.
+
+### Changelog
+
+* Fixed Dasharo zero-touch initial deployment on MSI PRO Z690-A (WIFI) DDR5.
```

Comment:
Fixed? In my opinion, we just added this functionality, the first release for this platform was released a month ago, in DTS v1.1.0 we did not dedicate support

Reply:
That sounds good for me

Reply:
@Pokisiekk added https://github.com/Dasharo/docs/pull/369/commits/1da7504b0ca37c218e721909b5a683068cc19ee4#diff-dcdc85616220542189f6c2c350d6a3d92405bdee1142a24b3c56bf3484994b07R39

Please resolve this conv